profiles/migrations/0007_profile_completeness.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 4.2.4 on 2024-01-04 10:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('profiles', '0006_profile_created_at_profile_updated_at'),
+    ]
+
+    def add_completeness_value(app, schema_editor):
+        Profile = app.get_model("profiles", "Profile")
+        Activity = app.get_model("profiles", "Activity")
+        Category = app.get_model("profiles", "Category")
+        for instance in Profile.objects.all():
+            instance.completeness = 0
+            if instance.banner_image:
+                instance.completeness += 100
+            if instance.logo_image:
+                instance.completeness += 1
+            if instance.region:
+                instance.completeness += 1
+            if Activity.objects.all().filter(profile=instance.id):
+                instance.completeness += 1
+            if Category.objects.all().filter(profile=instance.id):
+                instance.completeness += 1
+            instance.save()
```

Comment:
You already have two places with this code. It would be better to move this logic into function/module and use it here and in profile view